.github/workflows/test.yaml

```diff
@@ -48,7 +49,7 @@ jobs:
           if [ -n "$OPERATOR_INSTALL" ]; then
             for operator in $OPERATOR_INSTALL; do helm install --namespace default $operator charts/$operator/.; done
           fi
-          ct install --config .github/ct.yaml
+          ct install --config .github/ct.yaml --excluded-charts=everest
```

Comment:
Had to exclude it for now because this tool tries to deploy in a random namespace, but the chart fails if it is anything other than `everest-system`

At the same time, I could not figure out how to set a namespace only for specific charts.

Reply:
Maybe there can be a separate workflow for everest, since we'll need one anyway for running a manual release